docs/core/whats-new/dotnet-7.md

```diff
@@ -0,0 +1,181 @@
+---
+title: What's new in .NET 7
+description: Learn about the new features introduced in .NET 7.
+ms.date: 11/01/2022
+ms.topic: overview
+ms.author: gewarren
+author: gewarren
+---
+# What's new in .NET 7
+
+.NET 7 is the successor to [.NET 6](dotnet-6.md) and focuses on being unified, modern, simple, and *fast*. .NET 7 will be [supported for 18 months](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) as a standard-term support (STS) release (previously known as a *current* release).
+
+This article lists the new features of .NET 7 and provides links to more detailed information on each.
+
+<!--
+To find all the <https://learn.microsoft.com/dotnet> articles that have been updated for .NET 7, see [What's new in docs for .NET 7](dotnet-7-docs.md).
+-->
+
+## Performance
+
+Performance is a key focus of .NET 7, and all of its features are designed with performance in mind. In addition, .NET 7 includes the following enhancements aimed purely at performance:
+
+- On-stack replacement (OSR) is a complement to tiered compilation. It allows the runtime to change the code executed by a currently running method in the middle of its execution (that is, while it's "on stack"). Long-running methods can switch to more optimized versions mid-execution.
+- Profile-guided optimization (PGO) now works with OSR and is easier to enable (by adding `<TieredPGO>true</TieredPGO>` to your project file). PGO can also instrument and optimize additional things, such as delegates.
+- Improved code generation for Arm64.
+- [Native AOT](../deploying/native-aot/index.md) produces a standalone executable in the target platform's file format with no external dependencies. It's entirely native, with no IL or JIT, and provides fast startup time and a small, self-contained deployment. In .NET 7, Native AOT focuses on console apps and requires apps to be trimmed.
+- Performance improvements to the mono runtime, which powers Blazor WebAssembly, Android, and iOS apps.
```

Comment:
```suggestion
- Performance improvements to the Mono Runtime, which powers Blazor WebAssembly, Android, and iOS apps.
```

I believe Mono Runtime is a proper name, https://www.mono-project.com/docs/advanced/runtime/